### ISSUE TITLE
Fire suppression - combination of ethanol_pan and flow_rate doesn't work / delivers strange results

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1182]
```
Please complete the following lines...

Application Version: FDS 5.5.2
SVN Revision Number: 6706
Compile Date: Sep 07 2010
Operating System:  Windows (Vista/7 64bit)


Right now I'm experimenting with flame extinction by water mist.
I'm using FDS 5.5.2 as 64bit version.

I would like to use pans filled with fuel and nozzles that atomise water.

As start I used 2 FDS examples (ethanol_pan / flow rate) and combined them to one model.

First, why is ethanol_pan no longer a standard example although it is described in
the actual manual (page 74)?
Secondly it has to be adapted or it won't start with version 5.5.2 (the BDNF line with
mass loss has to be switched off).

Calculating ethanol_pan everything works fine. When adding the components of flow_rate
(water drops and the nozzles) no fire at all occurs. It calculates but the output is
zero. 

I checked several FDS versions, 5.3.1, 5.4.3 und 5.5.0 work, 5.5.1 and 5.5.2 not. But
with this versions the HHR shows strange results.
There are 4 nozzles that are activated every 5 seconds beginning with second 60. 
The influence of the water is visible in smokeview where the smoke is suppressed below
the level of the nozzles and the temperature is significantly going down. I expected
also a reduction for the HHR by the water drops. Instead of that the HHR rises up beginning
with the water activation (see attached picture).

I made some adjustments at the DUMP and the MISC line and changed some parameters for
the PROP ID concerning the nozzles but this shouldn't make the difference.

Also there is quite a mismatch in the HHR with water as second species in the modelling.
The unaltered Ethanol_pan test file has HHR of about 200kW.
My modelling with water delivers about 60kw and with water 90-100kW. How comes that?
There are no changes for the fuel part.
The water seems to influence the HRR even before it is activated.

So in summary, why is there no fire with version 5.5.2 and why is there the strange
HRR output with the older versions?
Failure in my modelling or in FDS itself?

My results so far:
Ethanol_pan works with 5.5.2.
Adding water drops won't work.
It works with 5.5.0 (and older) but delivers strange results.

Kind regards

Peter Möller
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-09-30 10:23:36

<hr>
- _Attachment: [fire_suppression_test.JPG](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1210/comment-0/fire_suppression_test.JPG)_
- _Attachment: [ethanol_pan_with_water_flow.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1210/comment-0/ethanol_pan_with_water_flow.fds)_

**Comment by gforney** [comment]
```
Thanks for the comments. The reason for removing this case from the V&V is that we have
found it to be very grid-sensitive and there are numerous issues related to physical
parameters like ABSORPTION_COEFFICIENT. We have not had time lately to work on this
because other issues are more pressing. However, we will try to address the problems
you have raised. Most importantly, the issue of the two files not working together.
```

Original issue reported on code.google.com by `mcgratta` on 2010-09-30 11:26:54

**Comment by gforney** [comment]
```
Thanks for the quick reply. Is there any example that addresses fire suppression combining
fuels and water?
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-09-30 11:37:16

**Comment by gforney** [comment]
```
As a note on your observation of different results with and without water, this is not
unexpected although the size of the difference is surprising.  In the currently formulation
of the background species, we do not include ambient water vapor UNLESS the input file
also contains sprinklers with water drops.  Therefore the two cases will have slightly
different specific heats, average molecular weights, and ambient oxygen mass fractions
and these will have some effect on the evaporation rate.  
```

Original issue reported on code.google.com by `drjfloyd` on 2010-09-30 12:00:50

**Comment by gforney** [comment]
```
Therefore I adjusted the HUMIDITY in both cases to 40% to minimise this effects. At
least I thought it would minimise them.
Besides that the fact that the HRR is rising after the activation of the nozzles is
far more astonishing to me.
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-09-30 12:16:00

**Comment by gforney** [comment]
```
From User's Guide 6.4:

HUMIDITY Relative humidity, in units of %. This need only be specified if there is
a source of water in the simulation other than the fire itself. Otherwise, water vapor
is not explicitly tracked. Default 40 %.

If you have no sprinklers, there is no species for water vapor and the keyword HUMIDITY
has no effect.  If you want equivalent sprinkler and non-sprinkler cases, you will
need to define the species WATER VAPOR either using SPEC or by putting in a sprinkler
but never allowing it to operate.
```

Original issue reported on code.google.com by `drjfloyd` on 2010-09-30 13:57:49

**Comment by gforney** [comment]
```
Thanks for clarifying that. You are absolutely right, I checked this already. Simulation
time is 100s. In one case the activation of the water drops is at 50s in the other
case at 550s. There has just the set point of the activation been altered, no other
changes in the files have been made.
Up to second 50 the to runs are identical, then the strange HHR rise can be seen (see
attached picture). Checked with 5.5.0 as higher versions do not work at all.


```

Original issue reported on code.google.com by `spamemailadresse` on 2010-09-30 15:23:49

<hr>
- _Attachment: [fire_suppression_test_2.JPG](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-1210/comment-6/fire_suppression_test_2.JPG)_

**Comment by gforney** [comment]
```
Yes, there is a bug in FDS 5.5.2. I am going to track it down.
```

Original issue reported on code.google.com by `mcgratta` on 2010-10-14 21:58:17

**Comment by gforney** [comment]
```
I fixed the bug that caused the fire to disappear when a sprinkler was added to the
case, but I have not fixed the problem where the HRR increases when the sprinklers
activate. I think that this is a problem with the evaporation model. The water droplets
cool the surface to more or less the ambient value, but the liquid still evaporates
at this temperature. There is nothing in the model that accounts for the water mixing
with the liquid fuel and changing its composition. I don't think that this situation
is going to be fixed soon. I think that the liquid fuel model is already problematic,
and the suppression makes it worse. 

I am going to put this "On Hold."
```

Original issue reported on code.google.com by `mcgratta` on 2010-10-27 17:32:12

**Comment by gforney** [comment]
```
Thanks for fixing the bug. 
Is there at the moment no possibility to mix a pan filled with burning liquid with
a water mist system to suppress the fire?
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-10-27 19:06:42

**Comment by gforney** [comment]
```
How can you be sure that the increase in HRR is a problem?
```

Original issue reported on code.google.com by `jukka.vaari@vtt.fi` on 2010-10-27 19:17:28

**Comment by gforney** [comment]
```
It is a problem to me as I have doubts about the calculations being realistic.
It seems strange to me that adding a cooling medium like water especially as water
mist causes an increase in the HRR. How can I be sure that the temperature reduction
is realistic if the HRR isn't?
Real fires with this constellation (pan with burning liquid and water mist) don't show
this behaviour. It least the ones I have seen.
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-10-27 19:30:20

**Comment by gforney** [comment]
```
I can tell you what is happening in the model. When the sprinklers activate, there is
a slight cooling of the liquid surface which would tend to decrease the mass loss rate
slightly. However, there is an increased velocity at the liquid surface that pushes
away the vaporized fuel, leading to an increased evaporation rate. 
```

Original issue reported on code.google.com by `mcgratta` on 2010-10-27 19:47:34

**Comment by gforney** [comment]
```
This sounds as changing the modelling parameters (e.g. more water, more sprinklers,
changing the water droplet speed, ... ) could lead to a drop? That simple or is the
problem within fds?
Does it make sense to execute further simulations changing parameters?
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-10-27 20:25:43

**Comment by gforney** [comment]
```
The problem, I believe, is the liquid fuel evaporation model. It is very sensitive to
both numerical and physical parameters. The cooling by liquid water droplets makes
the problem even more complicated. I think we need a re-formulation of the liquid fuel
evaporation model before considering the suppression by water.
```

Original issue reported on code.google.com by `mcgratta` on 2010-10-27 20:34:29

**Comment by gforney** [comment]
```
OK, I understand. Thanks for your response.
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-10-27 20:42:59

**Comment by gforney** [comment]
```
You need to decide what you want. You may want water spray to decrease HRR. The model
may or may not work to that effect, and your input parameters may or may not work to
that effect. In reality, water sprays may decrease or enhance HRR. Just browse some
literature, for example

Fire Safety Journal, Volume 27, Issue 1, July 1996, Pages 37-48 

shows enhancement.
```

Original issue reported on code.google.com by `jukka.vaari@vtt.fi` on 2010-10-27 20:44:21

**Comment by gforney** [comment]
```
Thanks for pointing that out. The literature and real burning test I have seen so far
don't mention or show that. I'll do further research.
```

Original issue reported on code.google.com by `spamemailadresse` on 2010-10-27 21:08:43

**Comment by gforney** [comment]
```
Just a quick reminder. Are there any new developments concerning this issue?
```

Original issue reported on code.google.com by `spamemailadresse` on 2011-06-08 09:03:17

**Comment by gforney** [comment]
```
We're still working on the next release. I'll try your case with the current version
to see if it is working better now.
```

Original issue reported on code.google.com by `mcgratta` on 2011-06-08 12:16:44

**Comment by gforney** [comment]
```
This is still a problem. We are planning, along with Simo Hostikka et al. at VTT, to
look more carefully at the assumptions we are making for liquid fuels. At the moment,
the burning rate is overly sensitive to the concentration of fuel gas in the first
grid cell above the liquid surface. We may need to overhaul the entire routine. Stay
tuned.
```

Original issue reported on code.google.com by `mcgratta` on 2011-06-10 21:21:01

**Comment by gforney** [comment]
```
Hello everybody. Any updates on this issue?
```

Original issue reported on code.google.com by `spamemailadresse` on 2011-11-25 14:29:31

**Comment by gforney** [comment]
```
No. There is no one available to work on it. 
```

Original issue reported on code.google.com by `mcgratta` on 2011-11-26 17:21:04

**Comment by gforney** [comment]
```
I am transferring this issue to Topi Sikanen.
```

Original issue reported on code.google.com by `mcgratta` on 2012-08-10 22:13:46

**Comment by gforney** [comment]
```
The case runs and the HRR decreases when sprinklers are turned on.  Water drops accumulate
on the surface of the fuel so  the mixing of drops with would need to be modeled. However
if there is need for such capability, we should probably open a feature request in
issue tracker. I'm marking this one as closed.
```

Original issue reported on code.google.com by `Topi.Sikanen` on 2012-08-15 19:39:08
